documentation/modules/metrics/proc_metrics-deploying-prometheus-operator.adoc

```diff
@@ -10,17 +10,19 @@ To deploy the Prometheus Operator to your Kafka cluster, apply the YAML bundle r
 
 .Procedure
 
-. Download the resource file from the repository and replace the example `namespace` with your own:
+. Download the `bundle.yaml` resources file from the repository and replace the example `namespace` with your own:
 +
 On Linux, use:
 +
-[source,shell,subs=+quotes]
-curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus-operator-deployment.yaml
+[source,shell,subs="+quotes,attributes+"]
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/{PrometheusSupport}/bundle.yaml | sed -e 's/namespace: .*/namespace: _my-namespace_/' > prometheus-operator-deployment.yaml
 +
 On MacOS, use:
 +
-[source,shell,subs=+quotes]
-curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml | sed -e '' 's/namespace: .*/namespace: _my-namespace_/' > prometheus-operator-deployment.yaml
+[source,shell,subs="+quotes,attributes+"]
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/{PrometheusSupport}/bundle.yaml | sed -e '' 's/namespace: .*/namespace: _my-namespace_/' > prometheus-operator-deployment.yaml
++
+Choose a release that is compatible with your version of Kubernetes.
```

Comment:
Not so simple :-( The Prometheus operator team provided me this link in the past for choosing a release https://github.com/coreos/kube-prometheus#kubernetes-compatibility-matrix but then if you are using OpenShift 3.11 for example the above URL is not valid anymore and the right bundle comes from https://github.com/openshift/prometheus-operator/tree/release-3.11. Anyway because we want to stay on vanilla Kubernetes here, I would add the link to the versions matrix. It's mentioned more or less here https://github.com/coreos/prometheus-operator#quickstart

Reply:
Of course not ;)
I'll add links to the Kubernetes compatibility matrix and the separate `openshift/prometheus-operator` repository. We mention OpenShift elsewhere in the docs, so I think it's OK to mention it here.

Reply:
I reworded as follows:

Use the  [Kubernetes compatibility matrix](https://github.com/coreos/kube-prometheus#kubernetes-compatibility-matrix) to help you choose a release that is compatible with your version of Kubernetes. If using OpenShift, specify a release of the [OpenShift fork](https://github.com/openshift/prometheus-operator) of the Prometheus Operator repository.

Reply:
I don't know about mentioning OpenShift or maybe we could have it as a note but not in the main text?

Reply:
I'll try it as a note. I think it's OK to mention it here; we mention `oc` and OpenShift as deployment options elsewhere in the documentation.

Reply:
I've rearranged the information again! The OpenShift option is now in a note. Please take another look.